exercises/knapsack/description.md

```diff
@@ -0,0 +1,24 @@
+In this exercise, lets try to solve a classic problem.
+
+Bob is a thief. After months of carefuly planning, he finally manages to crack the security systems of a high-class apartment. 
+
+In front of him are many items, each with a value (v) and weight (w). Bob of course wants to 
+maximize the total value he can get; he would gladly take all of the items if he could. 
+However, to his horror, he realized that the knapsack he carried with him can only hold so much weight(W).
+
+Given a knapsack with a specific capacity (W), can you help Bob determine the maximum value he can get from the items in the house? Note, Bob can only take only one of each item. 
```

Comment:
```suggestion
Given a knapsack with a specific capacity (W), help Bob determine the maximum value he can get from the items in the house. Note, Bob can only take only one of each item. 
```
The question is OK, My personal preference is to have a statement that states what needs to be done.  One only needs to answer `Yes` to the question.

Reply:
If you turn the sentence around you can eliminate the comma.
```
Help Bob determine the maximum value he can get from the items in the house given a knapsack with a specific capacity (W).
```

Reply:
Yup I think a statement is good as well!